Doc/library/stdtypes.rst

```diff
@@ -1491,8 +1491,7 @@ objects that compare equal might have different :attr:`~range.start`,
    sequence of values they define (instead of comparing based on
    object identity).
 
-.. versionadded:: 3.3
```

Comment:
Should this one stay? it describes an addition. The above bit thing was 3.3 a behavior change rather than API addition.

Do the docs folks have a preference for what order to list things in when a release has both added items and changed items for one doc area?  Or if added things can be listed in a combined changed entry?

Reply:
Here `versionadded` is changed to `versionchanged` (with the correction of the nested text), and that `versionchanged` is collapse with the previous `versionchanged` with the same version.

For the range object as a whole it is a change. If these were separate `attribute` directives for these attributes, they would have a `versionadded` sub-directive. This is how the added sub-items are usually documented if they cannot have their own `versionadded`.